frontend/packages/ux-editor/src/types/FormContainer.ts

```diff
@@ -14,3 +16,19 @@ export interface FormContainer {
   edit?: KeyValuePairs;
   [id: string]: any;
 }
+
+export type ContainerComponent =
+  | ComponentType.Accordion
+  | ComponentType.AccordionGroup
+  | ComponentType.ButtonGroup
+  | ComponentType.Group;
+
+export const containerComponentsWithValidChildrenMapping: Record<
+  ContainerComponent,
+  ComponentType[]
+> = {
+  [ComponentType.Accordion]: [ComponentType.Paragraph],
+  [ComponentType.AccordionGroup]: [ComponentType.Accordion],
+  [ComponentType.ButtonGroup]: [ComponentType.Button],
+  [ComponentType.Group]: Object.values(ComponentType),
+};
```

Comment:
`types`-mappen er kun for typer. Vi vil unngå å definere variabler her. Kan denne defineres et annet sted?